src/Jackett.Common/Indexers/TorrentSyndikat.cs

```diff
@@ -97,155 +93,178 @@ public TorrentSyndikat(IIndexerConfigurationService configService, WebClient w,
-                    var seeders = ParseUtil.CoerceInt(row.Children[7].TextContent);
-                    var link = new Uri(SiteLink + qLink.GetAttribute("href"));
-                    var files = ParseUtil.CoerceInt(sizeFileCountRowChilds[2].TextContent);
-                    var leechers = ParseUtil.CoerceInt(row.Children[8].TextContent);
-                    var grabs = ParseUtil.CoerceInt(row.QuerySelector("td:nth-child(7)").TextContent);
-                    var downloadVolumeFactor = globalFreeleech || row.QuerySelector("span.torrent-tag-free") != null
-                        ? 0 : 1;
-                    var size = ReleaseInfo.GetBytes(sizeFileCountRowChilds[0].TextContent);
                     var release = new ReleaseInfo
                     {
                         MinimumRatio = 1,
-                        MinimumSeedTime = 345600, //8 days
-                        Category = MapTrackerCatToNewznab(catStr),
-                        Link = link,
-                        Description = description,
-                        Title = qCommentLink.GetAttribute("title"),
-                        Comments = comments,
+                        MinimumSeedTime = 96 * 60 * 60,
+                        DownloadVolumeFactor = 1,
+                        UploadVolumeFactor = 1,
                         Guid = comments,
-                        PublishDate = pubDateUtc.ToLocalTime(),
-                        Imdb = longFromString,
-                        Size = size,
-                        Files = files,
+                        Comments = comments,
+                        Link = new Uri(SiteLink + "download.php?id=" + id),
+                        Title = row.Value<string>("name"),
+                        Category = MapTrackerCatToNewznab(row.Value<int>("category").ToString()),
+                        PublishDate = dateTime.AddSeconds(row.Value<long>("added")).ToLocalTime(),
+                        Size = row.Value<long>("size"),
+                        Files = row.Value<long>("numfiles"),
                         Seeders = seeders,
-                        Peers = leechers + seeders,
-                        Grabs = grabs,
-                        DownloadVolumeFactor = downloadVolumeFactor,
-                        UploadVolumeFactor = 1
+                        Peers = seeders + row.Value<int>("leechers"),
+                        Grabs = row.Value<int>("snatched"),
+                        Imdb = row.Value<long?>("imdbId"),
+                        TVDBId = row.Value<long?>("tvdbId"),
+                        TMDb = row.Value<long?>("tmdbId"),
                     };
+
+                    var descriptions = new List<string>();
+                    var title = row.Value<string>("title");
+                    var titleOrigin = row.Value<string>("title_origin");
+                    var year = row.Value<int?>("year");
+                    var pid = row.Value<int?>("pid");
+                    var poster = row.Value<string>("poster");
```

Comment:
The object ReleaseInfo has an attribute called `Banner` for this (if poster is a poster image URL). Use that and remove from description

Reply:
Thanks. Missed it yesterday since I used an old Jackett version…